sql_generators/active_users/templates/focus_android_view.sql

```diff
@@ -2,7 +2,8 @@ CREATE OR REPLACE VIEW
   `{{ project_id }}.{{ app_name }}.active_users_aggregates`
 AS
 SELECT
-  * EXCEPT (app_version),
+  * EXCEPT (app_version, app_name),
+  IF(app_name IN ('Focus Android Glean', 'Focus Android'), 'Focus Android', app_name) AS app_name,
```

Comment:
Is there a need to capture the case where app_name = 'Focus Android Glean BrowserStack'

Reply:
Given that this is a view update @wwyc let's merge and I've taken a note to ask Lucia about this when she is back.

Reply:
No, BrowserStack needs to be separated because is bot generated clients.